staticfiles/templates/contributors/radhika.html

```diff
@@ -0,0 +1,107 @@
+<link rel="stylesheet" href="http://radhikabhanu.in/home/assets/css/font-awesome.min.css">
+
+<body style="display: block;">
+
+
+    <div class="container">
+    	<div class="front">
+    	<div class="col-lg-6 col-lg-offset-3">
+    	
+    		<!-- ===== vCard Navigation ===== -->
+    		<div class="row w">
+    			<div class="col-md-4">
+    				<img class="img-responsive" src="https://lh3.googleusercontent.com/-XEtJ3ncHBqw/UOldOCWKz_I/AAAAAAAABEk/b5xkQyiNaro/w280-h280-p/meruch.jpg" alt="">
```

Comment:
you should link to this image locally, this is hosted on google. If you want to use the google image, please delete the one you uploaded above.